dbclient/MLFlowClient.py

```diff
@@ -179,8 +155,21 @@ def _create_experiment(self, experiment_str, id_map_writer, checkpointer, error_
             new_id = self.client.create_experiment(name, artifact_location, dict_tags)
             logging.info(f"Successfully created experiment with name: {name}. id: {new_id}")
         except RestException as error:
-            logging.info(f"create experiment failed for id: {id}, name: {name}. Logging it to error file..")
-            error_logger.error(error.json)
+            # If the resource already exists, then we can consider it successful and checkpoint it.
+            if error.json['error_code'] == 'RESOURCE_ALREADY_EXISTS':
```

Comment:
Should we also check here whether the experiment is a notebook experiment?

Reply:
We shouldn't. If even if is non-notebook experiment, we can't create the experiment anyway. If the experiment already exists in the same path, it's what it is.